main.go

```diff
@@ -53,8 +53,8 @@ func main() {
 	apiv1 := r.PathPrefix("/v1").Subrouter()
 	apiv1.Methods("GET").Path("/stars").HandlerFunc(GetStars)
 	apiv1.Methods("PUT").Path("/stars").HandlerFunc(UpdateStar)
-	apiv1.Methods("GET").Path("/comments/{itemID}").HandlerFunc(GetComments)
-	apiv1.Methods("POST").Path("/comments/{itemID}").HandlerFunc(CreateComment)
+	apiv1.Methods("GET").Path("/comments/{repo}/{chartName}").HandlerFunc(GetComments)
+	apiv1.Methods("POST").Path("/comments/{repo}/{chartName}").HandlerFunc(CreateComment)
```

Comment:
if the param was `/comments/{chartID}` you wouldn't need to concatenate them later right? I think we can assume that the client will call this correctly.

Reply:
nevermind, this isn't possible with mux